source/configuration_manual/authentication/authentication_mechanisms.rst

```diff
@@ -46,49 +46,48 @@ cleartext password.
+|                                                                              | Can be accessed via GSSAPI.                                              |                             |
++------------------------------------------------------------------------------+--------------------------------------------------------------------------+-----------------------------+
+| `GSSAPI <authentication-gssapi>`                                             | Kerberos v5 support.                                                     |                             |
++------------------------------------------------------------------------------+--------------------------------------------------------------------------+-----------------------------+
+| RPA                                                                          | Compuserve RPA authentication mechanism.                                 | .. versionremoved:: 2.3.13  |
+|                                                                              | Similar to DIGEST-MD5, but client support is rare.                       |                             | 
++------------------------------------------------------------------------------+--------------------------------------------------------------------------+-----------------------------+
+| ANONYMOUS                                                                    | Support for logging in anonymously. This may be useful if you're         |                             |
+|                                                                              | intending to provide publicly accessible IMAP archive.                   |                             |
++------------------------------------------------------------------------------+--------------------------------------------------------------------------+-----------------------------+
+| OTP                                                                          | One time password mechanisms.                                            |                             |
++------------------------------------------------------------------------------+--------------------------------------------------------------------------+-----------------------------+
+| EXTERNAL                                                                     | EXTERNAL SASL mechanism.                                                 |                             |
++------------------------------------------------------------------------------+--------------------------------------------------------------------------+-----------------------------+
+| OAUTHBEARER                                                                  | OAuth2 bearer authentication :rfc:`7628`.                                | .. versionadded:: 2.2.29    |
+|                                                                              | See :ref:`authentication-oauth2`.                                        |                             |
```

Comment:
Place of reference is inconsistent with other mechanisms above (those have it in the mechanism column if I understand the syntax correctly).